ecosystem/sep-0024.md

```diff
@@ -65,6 +65,14 @@ Note that Stellar accounts are either shared or they are not. This means anchors
 
 Note that both the source account of a withdrawal payment and the destination account of a deposit can be different than the account authenticated via SEP-10.
 
+## Asset Exchanges
+
+This protocol was originally designed to provide conversions between off-chain and on-chain equivalent assets – for instance `BRL <> BRLT`, `USD <> USDC`, `NGN <> NGNT`, etc. – but it has been updated to make exchanges between off-chain and on-chain non-equivalent assets like `BRL <> USDC`.
```

Comment:
Making exchanges between off-chain and on-chain non-equivalent assets seem to overlap with SEP-38. I am not sure if we should update sep-24 or update both sep-24 and sep-38 together.

Reply:
SEP-38 doesn't have any information on how the quotes returned from its endpoints are used in other protocols, so I don't think we need any adjustments there.

However, there are functionality gaps between SEP-6 and SEP-24 in terms of how they support non-equivalent assets. For example, you cannot provide a SEP-38 quote ID to SEP-24's endpoints like you can with SEP-6. Client applications also cannot fetch `GET /quote` information for the exchange of assets within a SEP-24 transaction, although they can infer most of it from the info returned in `GET /transaction(s)`.

All this is to say that we could further integrate SEP-38 into SEP-24, but the changes made here are the bare-bones needed to facilitate this use case. cc @leighmcculloch